python/cac_tripplanner/templates/partials/ios-weblinks.html

```diff
@@ -0,0 +1,5 @@
+// Keep iOS in standalone mode when following links
```

Comment:
Oops, that's a reply to the line comment.  I wish it would let you reply to the overall review comment...